README.md

```diff
@@ -15,30 +15,9 @@ All the official documentation on the Salesforce CLI Scanner plug-in is hosted o
 ### Document Link
 https://forcedotcom.github.io/sfdx-scanner/
 
+### Contributing
```

Comment:
Could this be ### Building/Contributing? It seems that `Contributing` is typically more legalise than instructions, users might not know to look here.

Reply:
Updated in a newer commit